### ISSUE TITLE
FE: Insufficient contrast ratio

### ISSUE THREAD

**Issue by rsteckline** [2308-Bread/rails_bread_fe#58]
![Screenshot 2024-04-24 at 1 24 30 PM](https://github.com/2308-Bread/rails_bread_fe/assets/135551833/d3fbc6ab-1af0-4c83-8f5d-cc6bdda39615)

Need to adjust color for accessibility issues
